### DISCUSSION TITLE
Repo moved, tags out of date?

### DISCUSSION THREAD

**Discussion by miikalo** [openupm/openupm#1849] [Q&A]
Hello,

I maintain a package that was moved to a different repository and the package names have now changed as well. Updating the project location to the new repository was simple, but all the tags still have the old addresses and names, and if I understood openupm correctly, that's where openupm looks for the package information in package.json.

What's the recommended way to update the tags?

**Comment by favoyang** [comment]
Hi @miikalo,

When you want to change the package name, you need to update the YAML file name and the package name together as a PR. Then the build pipelines will treat it as a new package.
- All existing tags will be rebuild but failed because of the changed package name. You can avoid this by using the "minVersion" field.
- You need to bump the version to prepare a Git tag for the new package name.
- Old packages will remain on the registry but disconnect with the new package page.

That's the current behavior. You can bring your idea if you think we can improve this process.
